src/DynamoPackages/Package.cs

```diff
@@ -338,7 +340,7 @@ internal bool ContainsFile(string path)
             return Directory.EnumerateFiles(RootDirectory, "*", SearchOption.AllDirectories).Any(s => s == path);
         }
 
-        internal bool InUse(DynamoModel dynamoModel)
+        public bool InUse(DynamoModel dynamoModel)
```

Comment:
these methods are at Package level, and this type is in DynamoPackages project which we are not going to remove. So making them public I thought will make more sense than implementing in our project again which I guess not possible.